src/app.js

```diff
@@ -1 +1,32 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const createCopy = (pathToFile, pathToCopyFile) => {
+  let oldFile;
+
+  if (pathToFile === pathToCopyFile) {
+    return;
+  }
+
+  try {
+    const data = fs.readFileSync(pathToFile, 'utf8');
+
+    oldFile = data;
+  } catch (error) {
+    throw new Error('Wrong path to file');
+  }
+
+  try {
+    fs.writeFileSync(pathToCopyFile, oldFile);
```

Comment:
instead of using `readFileSync` and write `writeFileSync`, you can use just `copyFileSync`